scripts/config/templates/nginx.conf.erb

```diff
@@ -163,6 +163,12 @@ http {
     proxy_set_header X-Forwarded-Proto  $scheme;
   }
 
+  # Redirect our locations pages to the new url structure
+  location ~ "(*UTF8)^\/locations\/[A-z][A-z]\/[A-z\-\/]*$" {
```

Comment:
Are there any other matches in this file for /locations pages?  enters this block that means it will not enter others. They should be unified into the same block

Reply:
There are no other matches for `/locations` in this file. This will only take in urls with state being only 2 characters too.

Reply:
ah I see now